CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 ## Unreleased
 
-- Update the Guzzle tracing middleware to meet the [expected standard](https://develop.sentry.dev/sdk/features/#http-client-integrations) (#1234)
+- Update Guzzle tracing middleware to meet the [expected standard](https://develop.sentry.dev/sdk/features/#http-client-integrations) (#1234)
 - Add `toArray` public method in `PayloadSerializer` to be able to re-use Event serialization
+- The `withScope` methods now return the callback's return value (#1263)
```

Comment:
```suggestion
- The `withScope` methods now returns the callback's return value (#1263)
```

Reply:
@Jean85 Methods is plural since there are three implementations, so return should be singular, but I can change it if you want.

Reply:
No you're right.